Source/Timeline/EventView.swift

```diff
@@ -102,7 +102,7 @@ open class EventView: UIView {
     context.setStrokeColor(color.cgColor)
     context.setLineWidth(3)
     context.translateBy(x: 0, y: 0.5)
-    let x: CGFloat = 0
+    let x: CGFloat = frame.width - 3 // 3 is the line width
```

Comment:
1. Line Width should be 1 for now, it looks weird when `3` is used
2. UI layout direction is not respected here, which leads to a broken layout in LTR languages (e.g. English) as shown on the screenshot
![Simulator Screen Shot - iPhone 12 Pro - 2020-12-30 at 02 58 22](https://user-images.githubusercontent.com/8013017/103321751-285bb200-4a4c-11eb-858b-75bb970b0055.png)

Reply:
Hmm. that's weird. I'll fix that

Reply:
line 103 says: ```context.setLineWidth(3)```. Should I change that to 1?